lib/config/param.c

```diff
@@ -626,7 +627,7 @@ param_default_validator(const struct param_spec *ps, const void *value)
             case PARAM_TYPE_U64:                                                                  \
                 assert(ps->ps_size == sizeof(uint64_t));                                          \
                 if (tmp < 0 || tmp > (double)UINT64_MAX) {                                        \
-                    CLOG_ERR(                                                                         \
+                    CLOG_ERR(                                                                     \
                         "Number of bytes of %s is not within the bounds of an unsigned 64-bit "   \
                         "integer",                                                                \
                         ps->ps_name);                                                             \
```

Comment:
It would be more readable if you used a better macro arg name, e.g. **_units** or something.  Also, you can put a space on either side of **##** to make it more readable.

Reply:
Changed the macro arg name, but clang-format won't allow a space around ##. Maybe there is a setting for investigation.